bindings/py/cpp_src/bindings/math/py_SparseMatrix.cpp

```diff
@@ -177,8 +177,8 @@ namespace nupic_ext
         sm.def("setToZero", &SparseMatrix32_t::setToZero);
         sm.def("setFromOuter", &SparseMatrix32_t::setFromOuter, py::arg("x"), py::arg("y"), py::arg("keepMemory") = false);
         sm.def("setFromElementMultiplyWithOuter", &SparseMatrix32_t::setFromElementMultiplyWithOuter);
-        sm.def("setRowFromDense", &SparseMatrix32_t::setRowFromDense);
```

Comment:
why is setRowFromDense removed? 
...but we can hope to delete this class soonish anyways..

Reply:
> why is setRowFromDense removed?

No, this is only one of the overloaded calls.  The sources has two setRowFromDense() calls, each with different signatures.  One accepting a vector and one taking an iterator.  I only wanted to expose the one with the vector (see line 606).  The other one I changed the name so that it did not have to be listed.